src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -293,7 +293,6 @@ void PutFromRevision()
                 }
 
                 flags |= DocumentFlags.Revision;
-                var data = context.ReadObject(document, id);
```

Comment:
I don't see why this would cause any issues, but I think that a recent change of mine might have caused it, want to go over this with you tomorrow.